core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/domain/common/accessor/MemberAccessorFactory.java

```diff
@@ -5,41 +5,18 @@
 import java.lang.reflect.Member;
 import java.lang.reflect.Method;
 import java.lang.reflect.Modifier;
-import java.util.Map;
 
 import org.optaplanner.core.api.domain.common.DomainAccessType;
 import org.optaplanner.core.api.solver.SolverFactory;
 import org.optaplanner.core.impl.domain.common.ReflectionHelper;
 import org.optaplanner.core.impl.domain.common.accessor.gizmo.GizmoMemberAccessorFactory;
 
-public class MemberAccessorFactory {
+public final class MemberAccessorFactory {
```

Comment:
So we have multiple kinds of MAFs, and they are in no relationship with one another? This is confusing me.